2019_BOSC/abstract/abstract.txt

```diff
@@ -1,11 +1,11 @@
 Biopython is a long-running distributed collaborative effort, a freely available Python library for biological computation. We summarise recent project news, and look ahead.
 
-Releases 1.73 (December 2018) and 1.74 (expected May/June 2019) had incremental improvements, especially on the in-line API documentation. Every public API should be documented this year.
+Releases 1.73 (December 2018) and 1.74 (expected May/June 2019) had incremental improvements, especially on the inline API documentation. Every public API should be documented this year.
```

Comment:
The "on" here sounds a bit strange, maybe "concerning"?

Reply:
Maybe "especially on the inline API documentation."?

Note I had to really cut this down for the 200 word plain text abstract, but we can keep the long version for the BOSC abstract booklet.

Reply:
or "especially in the API documentation"?

Reply:
Done in c2d23169cb400d8c323dd8a466a349f93d55e82b

Reply:
Sorry, comments passed each other. Python's API documentation is literally in the code itself ( do you know the term literate programming?), rather than in separate files. Thus "inline" was deliberate, but perhaps not as clear?

Reply:
No worries; I think the current version if fine, although I don't think the inline adds much for the purpose of this abstract. Nice job overall, I know how challenging word limits are!